common/types/block_scale.go

```diff
@@ -178,6 +178,13 @@ func (t *BlockContextualValidity) EncodeScale(enc *scale.Encoder) (total int, er
 		}
 		total += n
 	}
+	{
```

Comment:
weird that we generate scale for this type, can you please add a whitelist for Block and what else is needed?